docs/source/parser.rst

```diff
@@ -0,0 +1,104 @@
+Parser
+------
```

Comment:
Why its `-----` here and everywhere else it is `=====`? :-P
It wont matter AFAIK as the 'levels' are defined by succession of headings in each document, but keeping a convention would be good. (https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings)